scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -413,25 +440,25 @@ head( rowData(merged_sce) )
 
 So far, we've created a `merged_sce` object which is (almost!) ready for integration.
 
-The integration methods we'll be using here actually perform batch correction on a reduced dimension representation of the normalized gene expression values.
-`fastMNN` and `harmony` specifically use PCA for this, but be aware that different integration methods may use other kinds of reduced dimensions!
-The main reason for using reduced dimensions is efficiency.
+The integration methods we'll be using here actually perform batch correction on a reduced dimension representation of the normalized gene expression values, which is more efficient.
+`fastMNN` and `harmony` specifically use PCA for this, but be aware that different integration methods may use other kinds of reduced dimensions.
 
 You'll notice that the merged SCE object object already contains PCA and UMAP reduced dimensions, which were calculated during our pre-processing:
 
-```{r merged sce view pca}
+```{r merged_sce_reddim, live = TRUE}
+# Print the reducedDimNames of the merged_sce
 reducedDimNames(merged_sce)
 ```
 
 These represent the original dimension reductions that were performed on _each individual SCE_ before merging, but we actually need to calculate PCA (and UMAP for visualization) from the merged object directly.
 
 Why can't we use the sample-specific PCA and UMAP matrices? 
 Part of these calculations themselves involves scaling the raw data to center the mean.
-When samples are separately centered but plotting together, you will see samples "overlapping" in space, but this placement is an artifact of the individual centering.
+When samples are separately centered but plotting together, you will see samples "overlapping" in space, but this placement is actually just an artifact of the individual centering.
 In addition, the mathematical relationship between the original expression data and reduced dimension version of that data will differ across samples, meaning we can't interpret them all together.
 To see how this looks, let's look at the UMAP when calculated from individual samples:
 
-```{r individual UMAPs}
+```{r individual_UMAPs}
```

Comment:
I think I might reverse approach you took here. I'd want to show the UMAP plot live the first time, and then maybe not show every followup live, as they get repetitive.

Reply:
This is better!!